fastembed/text/pooled_embedding.py

```diff
@@ -50,6 +50,10 @@ def _get_worker_class(cls) -> Type[TextEmbeddingWorker]:
 
     @classmethod
     def mean_pooling(cls, model_output: np.ndarray, attention_mask: np.ndarray) -> np.ndarray:
+        if model_output.ndim == 2:  # (batch, embedding_dim)
+            seq_length = attention_mask.shape[1]
+            # (batch, seq_length, embedding_dim)
+            model_output = np.tile(np.expand_dims(model_output, axis=1), (1, seq_length, 1))
```

Comment:
is it actually required? how did it work before?

Reply:
Mean pooling is applied to token embeddings, however, it seems that jina provides sentence embeddings straightaway, so pooling is not required.
Also I don't really get the point about 1D and 2D
1) I don't understand why images are related to the text embedding class
2) As it have mentioned, since mean pooling is applied to token embeddings, expected shape is (batch_size, num_tokens, dim)